src/components/test/Dialog-test.js

```diff
@@ -118,18 +118,15 @@ describe('Dialog', () => {
       );
     });
 
-    it('focuses the dialog if `initialFocus` ref is `null`', () => {
-      const wrapper = mount(
-        <Dialog initialFocus={{ current: null }} title="My dialog">
+    it('does not handle focus if `initialFocus` is set to `null`', () => {
+      mount(
+        <Dialog initialFocus={null} title="My dialog">
           <div>Test</div>
         </Dialog>,
         { attachTo: container }
       );
 
-      assert.equal(
-        document.activeElement,
-        wrapper.find('[role="dialog"]').getDOMNode()
-      );
+      assert.equal(document.activeElement, document.body);
```

Comment:
This assumes that `document.activeElement` was `document.body` prior to running the test. A safer approach would be to capture the value before mounting and check that it doesn't change.

Reply:
Good catch. Will update.